src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -369,13 +402,16 @@ ELSE NULL
 
                             // Identify IDENTITY columns, as well as SERIAL ones.
                             var identityChar = record.GetValueOrDefault<char>("attidentity");
+
                             if (identityChar == 'a')
                                 column[NpgsqlAnnotationNames.ValueGenerationStrategy] = NpgsqlValueGenerationStrategy.IdentityAlwaysColumn;
+
                             else if (identityChar == 'd')
                                 column[NpgsqlAnnotationNames.ValueGenerationStrategy] = NpgsqlValueGenerationStrategy.IdentityByDefaultColumn;
+
                             else if (SerialTypes.Contains(systemTypeName) &&
-                                column.DefaultValueSql == $"nextval('{column.Table.Name}_{column.Name}_seq'::regclass)" ||
-                                column.DefaultValueSql == $"nextval('\"{column.Table.Name}_{column.Name}_seq\"'::regclass)")
+                                     column.DefaultValueSql == $"nextval('{column.Table.Name}_{column.Name}_seq'::regclass)" ||
+                                     column.DefaultValueSql == $"nextval('\"{column.Table.Name}_{column.Name}_seq\"'::regclass)")
```

Comment:
Decrease indentation. There is `a && b || c`, not `a && (b || c)`.

Reply:
Oops, that's some autoformat gone awry.